modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -222,6 +194,37 @@ public Page<ObjectDefinition> getObjectDefinitionsPage(
 					GetterUtil.getLong(document.get(Field.ENTRY_CLASS_PK)))));
 	}
 
+	@Override
+	public Page<ObjectDefinition>
+			getObjectFolderByExternalReferenceCodeObjectDefinitionsPage(
+				String externalReferenceCode, String search,
+				Pagination pagination)
+		throws Exception {
```

Comment:
I analyzed `getObjectDefinitionByExternalReferenceCodeObjectActionsPage` and `getObjectDefinitionByExternalReferenceCodeObjectFieldsPage`.

I see that we use the notation @NestedField(). Could you analyze if it would make sense to use in this method?

Reply:
Resolved.